File: BackEndApplication/KanBanBoardApplication/EmployeeTask/src/main/java/com/EmployeeTask/EmployeeTask/domain/Project.java
package com.EmployeeTask.EmployeeTask.domain;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;
import java.util.List;
import java.util.Set;

@AllArgsConstructor
@NoArgsConstructor
@ToString
@Data

public class Project {
    private int projectID;
    private String projectName;
    private String projectDescription;
    private String createdBY;
    private String projectCreatedDate;
    private String projectEndDate;
    private List<Task> taskList;
    private Set<String> employeeList;
}

File: Applications/Application/src/app/Model/Project.ts
import { ITask } from "./task";

export interface Project {
    projectID: number;
    projectName: string;
    projectDescription: string;
    createdBY: string;
    projectCreatedDate: string;
    projectEndDate: string;
    taskList: ITask[];
    employeeList: string[];
  }